### PR TITLE
Add Nyquist Rev 3

### PR DESCRIPTION
**Pull request by nooges**

### CODE REVIEW

File: keyboards/nyquist/rules.mk [hunk 1]
```diff
@@ -41,9 +41,13 @@ F_USB = $(F_CPU)
 
 # Bootloader
 #     This definition is optional, and if your keyboard supports multiple bootloaders of
-#     different sizes, comment this out, and the correct address will be loaded 
+#     different sizes, comment this out, and the correct address will be loaded
 #     automatically (+60). See bootloader.mk for all options.
-BOOTLOADER = caterina
+ifeq ($(strip $(KEYBOARD)), nyquist/rev3)
+    BOOTLOADER = dfu
```
Comment (mechmerlin) [inline keyboards/nyquist/rules.mk]: yeessss, a decent bootloader. 

Can someone actually still put a pro micro in though?

Comment (nooges) [reply keyboards/nyquist/rules.mk]: Nyquist 3 actually has an on-board controller, so someone would have to go through a considerable effort to put a Pro Micro on there 😄

File: keyboards/nyquist/rev2/config.h [hunk 2]
```diff
@@ -15,8 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef REV2_CONFIG_H
-#define REV2_CONFIG_H
+#pragma once
 
 #include QMK_KEYBOARD_CONFIG_H
```
Comment (drashna) [inline keyboards/nyquist/rev2/config.h]: This actually doesn't do anything and can be removed safely. 

```suggestion
```

File: keyboards/nyquist/rev3/config.h [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include QMK_KEYBOARD_CONFIG_H
```
Comment (drashna) [inline keyboards/nyquist/rev3/config.h]: This doesn't actually do anything, and can be safely removed, actually.

```suggestion
```

**Comment by drashna** [comment]
Travis CI errors are unrelated. 

Specifically @mtdjr's firmware is 98 bytes over, and @Xyverz's keymap has RGB (ws2812) related issues (from the community layout)
(users tagged for a heads up, and changes aren't necessary)

**Comment by nooges** [comment]
Converted Nyquist over to split_common

**Comment by drashna** [comment]
Thanks!

**Pull request merged**